docker-compose.yml

```diff
@@ -57,6 +47,8 @@ services:
     image: nginx:alpine
     depends_on:
       - django
+    ports:
```

Comment:
We have a reverse proxy in front so we don't expose the internal 8000 port.

Reply:
Does api calls works without this port open ?

Reply:
fixed